react/src/Attachment/Attachment.tsx

```diff
@@ -119,6 +119,9 @@ export type AttachmentProps<Multiple extends boolean> =
     onRejection?: (rejections: FileRejection[]) => void
   } & AttachmentValueProp<Multiple>
 
+/**
+ * @deprecated Use `FileInput` component instead, as that is much more customisable.
+ */
```

Comment:
Does this mean next time, folks who want the super default option got to include all those `FileInput.*` elements?